.travis_publish

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+
+# save current path
+REPO_PATH=$PWD
+
+# Save some useful information
+SHA=`git rev-parse --verify HEAD`
+
+# Clone the existing gh-pages for this repo into $HOME/gh-pages/
+# Create a new empty branch if gh-pages doesn't exist yet (should only happen on first deply)
+cd $HOME
+git config --global user.email "travis@travis-ci.org"
+git config --global user.name "travis-ci"
+git clone --quiet --branch=master https://${GH_PAGES_DEPLOY}@github.com/camel-tooling/camel-lsp-client-eclipse-update-site.git gh-pages > /dev/null
```

Comment:
why there are some gh-pages mentioned here?

Reply:
I just name the clone folder gh-pages.